packages/forms-web-app/__tests__/unit/controllers/projects/documents/filters/dates/config/expected-from-date-is-after-to-date-dates-filter-obj.js

```diff
@@ -0,0 +1,96 @@
+const expectedFromDateIsAfterToDateDatesFilterObj = {
```

Comment:
these are not config and should not be in a config dir. 

these could be called fixtures